src/services/pcn-packetcapture/src/cbpfTOc.cpp

```diff
@@ -0,0 +1,490 @@
+#include <iostream>
```

Comment:
This file is the core of your work.
Please:
  (a) add some reference (link) to Cloudfare, where you get some ideas (people get upset when you don't give them credits)
  (b) add some comments. It may not be trivial to understand what's going on in this file.